nifi-nar-bundles/nifi-jms-bundle/nifi-jms-processors/src/main/java/org/apache/nifi/jms/processors/ConsumeJMS.java

```diff
@@ -54,8 +61,31 @@
 @SeeAlso(value = { PublishJMS.class, JMSConnectionFactoryProvider.class })
 public class ConsumeJMS extends AbstractJMSProcessor<JMSConsumer> {
 
+    static final AllowableValue AUTO_ACK = new AllowableValue(String.valueOf(Session.AUTO_ACKNOWLEDGE),
```

Comment:
It's a bit confusing to see "Default value: 2" when I hover over the tooltip for the Acknowledgement Mode property.  It looks like the serialized values will be "1", "2", or "3".  Perhaps this is more of a problem in the UI rather than AllowableValue definition.